clients/src/main/java/org/apache/kafka/common/config/TopicConfig.java

```diff
@@ -75,16 +75,16 @@ public class TopicConfig {
         "\"delete\" retention policy. This represents an SLA on how soon consumers must read " +
         "their data. If set to -1, no time limit is applied.";
 
-    public static final String REMOTE_LOG_STORAGE_ENABLE_CONFIG = "remote.storage.enable";
+    public static final String REMOTE_LOG_STORAGE_ENABLE_CONFIG = "remote.log.storage.enable";
```

Comment:
Topic configurations do not include "log". That is why we decided not to have for the newly introduced configs. Same for other configs mentioned in this PR.